podman_tutorial.md

```diff
@@ -0,0 +1,153 @@
+                                    gpgme-devel glib2-devel device-mapper-devel libseccomp-devel \
+                                    atomic-registries iptables skopeo-containers containernetworking-cni
+```
+### Building and installing podman
+```
+# git clone https://github.com/projectatomic/libpod/ ~/src/github.com/projectatomic/libpod
+# cd !$ (or cd src/github.com/projectatomic/libpod
+# make
+# sudo make install PREFIX=/usr
+# sudo ln -s /usr/libexec/crio /usr/local/libexec/
+```
+
+<!-- (
+## Install podman on Ubuntu
+
+The default Ubuntu cloud image size will not allow for the following exercise to be done without increasing its
+capacity.  Be sure to add at least 5GB to the image. Instructions to do this are outside the scope of this
+tutorial.
+
+```
+# sudo apt-get update
+# sudo apt-get install libdevmapper-dev libglib2.0-dev libgpgme11-dev golang libseccomp-dev \
+                        go-md2man libprotobuf-dev libprotobuf-c0-dev libseccomp-dev
+# mkdir -p ~/src/github.com/projectatomic/
+# cd ~/src/github.com/projectatomic/
+# git clone https://github.com/projectatomic/libpod/
+#
+# cd libpod
+# make
+# sudo make install PREFIX=/usr
+# sudo mkdir -p /etc/containers
+# sudo bash -c 'cat <<EOF > /etc/containers/registries.conf
+registries.search
+registries = ['docker.io', 'registry.fedoraproject.org']
+EOF
+'
+# sudo bash -c 'cat <<EOF > /etc/containers/policy.json
+ {
+     "default": [
+         {
+             "type": "insecureAcceptAnything"
+         }
+     ],
+     "transports":
+         {
+             "docker-daemon":
+                 {
+                     "": [{"type":"insecureAcceptAnything"}]
+                 }
+         }
+ }
+ EOF
+'
+# git clone https://github.com/containernetworking/plugins.git ~/src/github.com/containernetworking/plugins
+# cd ~/src/github.com/containernetworking/plugins
+# ./build
+# sudo mkdir -p /usr/libexec/cni
+# sudo cp bin/* /usr/libexec/cni
+# git clone https://github.com/opencontainers/runc.git ~/src/github.com/opencontainers/runc
+# GOPATH=~/ make static BUILDTAGS="seccomp selinux"
+# sudo cp runc /usr/bin/runc
+# sudo mkdir -p /usr/local/libexec
+# sudo ln -s /usr/libexec/crio /usr/local/libexec/
+```
+
+) -->
+## Familiarizing yourself with podman
+
+### Running a sample container
+This sample container will run a very basic httpd server that serves only its index
+page.
+```
+# sudo podman run -dt -e HTTPD_VAR_RUN=/var/run/httpd -e HTTPD_MAIN_CONF_D_PATH=/etc/httpd/conf.d \
+                    -e HTTPD_MAIN_CONF_PATH=/etc/httpd/conf \
+                    -e HTTPD_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/httpd/ \
+                    registry.fedoraproject.org/f26/httpd /usr/bin/run-httpd
+```
+Note the Container ID returned by that command.
+
+### Listing running containers
+Once you have created or run a container, you can list the known containers with the *ps* command.
```

Comment:
Perhaps:  "Once a container is created or running,"

Reply:
that's passive voice .... a no-no :)

Reply:
Passive voice is my English weakness.  Regardless, a container can't be run unless it's created, that's what I badly was trying to get to.  Perhaps you could just drop the bit before the comma and maybe go with "The *ps* command is used to list created and running containers."

Reply:
SGTM

Reply:
yup